content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -245,23 +242,40 @@ Filled form event contains data from a form (prechat or postchat survey).
 
 Message event contains text message to other chat users.
 
-| Field            | Req./Opt. |  Notes                                                              |
-| ---------------- | :-------: | ------------------------------------------------------------------: |
-| `created_at`     | required  | Date & time format with a resolution of microseconds, `UTC string`. |
-| `recipients`     | required  | Possible values: `all` (default), `agents`                          |
-| `text`           | required  | Max. raw text size is 16 KB (one UTF-8 char like emoji 😁 can use up to 4 B); to send more, split text into several messages. |
-| `type`           | required  | `message`                                                           |
-| `custom_id`      | optional  |                                                                     |
-| `postback`       | optional  | Appears in the message event only when triggered by a rich message. |
-| `postback.type`  | optional  | Required only if `postback.value` is present.                       |
-| `postback.value` | optional  | Required only if `postback.type` is present.                        |
-| `properties`     | optional  | The [properties](#properties) object                                |
+### Requests
+
+| Field                 | Required| Data type | Notes                                                               |
+| --------------------- | ------- | --------- | ------------------------------------------------------------------  |
+| `custom_id`           | no      | `string`  |                                                                     |
+| `text`                | yes     | `string`  | Max. raw text size is 16 KB (one UTF-8 char like emoji 😁 can use up to 4 B); to send more, split text into several messages. |
+| `type`                | yes     | `string`  | `message`                                                           |
+| `recipients`          | yes     | `string`  | Possible values: `all` (default), `agents`                          |
+| `properties`          | no      | `object`  | [Properties](#properties)                                           |
+
+### Responses
+
+| Field                   | Returned    | Notes                                                                                      |
+| ------------------------| ----------- | ------------------------------------------------------------------------------------------ |
+| `id`                    | always      | Generated server-side                                                                      |
```

Comment:
I think `Generated server-side` notes are no longer needed when request and response fields are listed separately

Reply:
true